docs/05_best-practices/08_abi/01_abi-code-generator-attributes-explained.md

```diff
@@ -0,0 +1,88 @@
+## ABI/Code generator attributes explained
+The new ABI generator tool uses C++11 or GNU style attributes to mark ```actions``` and ```tables```.
```

Comment:
You can just use single "`" characters for single line monospace

Reply:
yes I guess we can do this here. I hope the original author doesn't mind :)